puzzle_generator/bytes_utils.py

```diff
@@ -5,14 +5,13 @@
 
 
 def int_to_bytes(in_value: int) -> bytes:
-    assert in_value >= 0  # nosec B101
+    if in_value < 0:
+        raise ValueError("in_value must be a positve integer")
```

Comment:
```suggestion
        raise ValueError("in_value must be non-negative")
```

What do you think about _covering_ this line by a test like:
```python3
def test_int_to_bytes_raises_when_input_is_negative():
    with pytest.raises(ValueError, match="in_value must be non-negative"):
        bu.int_to_bytes(-1)
```
This test should be added to [`test_bytes_utils.py`](https://github.com/vil02/puzzle_generator/blob/0353bba4f79b5780099e2135f77eacd0841f2cca/tests/test_bytes_utils.py)